sync/block_listener.go

```diff
@@ -10,9 +10,10 @@ import (
 	"time"
 )
 
+//BlockListener Listens to blocks propagated in gossip
```

Comment:
I believe in order for all the tools like godoc to work with the comments you need a single space between the comment token and the comment itself.

```suggestion
// BlockListener Listens to blocks propagated in gossip
```

Reply:
no this works, i checked

Reply:
what's the convention? space?